### PR TITLE
added liquibase script: event, image, event_type

### PR DESCRIPTION
**Pull request by VadimBlagiy**

### CODE REVIEW

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml [hunk 1]
```diff
@@ -0,0 +1,150 @@
+databaseChangeLog:
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml:1]: Нужно поменять порядок создания таблиц, в начале делаем таблицы, на которые нет зависимостей (image, event_type), потом уже event ( c зависимостями на image и event_type)

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+              - column:
+                  name: EVT_ID
+                  type: INT
+                  constraints:
+                    foreignKeyName: fk_EVENT_contact_EVENT_TYPE
+                    references: EVENT(EV_ID)
+              - column:
+                  name: EV_SHORT_DESCRIPTION
+                  type: VARCHAR(250)
+                  constraints:
```
Comment (ikantsevich) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml:105]: Do we sure that all columns are not null?

**Comment by VadimBlagiy** [comment]
- column:
                  name: EV_SHORT_DESCRIPTION
                  type: VARCHAR(250)
                  constraints:
                    nullable: false             

I think so. This is  really necessary field to fill! It shouldn't be left empty. Or am I wrong?

File: sandbox-backend/src/main/resources/db/changelog/changes/create-demo-table.yaml [hunk 3]
```diff
@@ -15,3 +15,14 @@ databaseChangeLog:
               - column:
                   name: SOME_FIELD
                   type: VARCHAR(255)
+  - changeSet:
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/create-demo-table.yaml:18]: Please remove this changeset. I've already merged similar that should drop demo table

File: sandbox-web/src/main/resources/application.properties [hunk 4]
```diff
@@ -1,4 +1,4 @@
-server.port=8080
```
Comment (Clasker1) [inline sandbox-web/src/main/resources/application.properties:1]: Lets use 8085, or just remove this change from PR

**Review by Clasker1 — CHANGES_REQUESTED** [review]

File: sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml [hunk 5]
```diff
@@ -0,0 +1,124 @@
...
+                  name: IMG_NAME
+                  type: VARCHAR(20)
+              - column:
+                  name: IMG_EXT
+                  type: VARCHAR(15)
+              - column:
+                  name: IMG_SIZE
+                  type: INT
+              - column:
+                  name: IMG_CREATED_AT
```
Comment (Clasker1) [inline sandbox-backend/src/main/resources/db/changelog/changes/2021.03.26_create_table_blagiy.yml:30]: Its good practice to use ```defaultValue: now()``` for create at fields.

**Review by Clasker1 — CHANGES_REQUESTED** [review]

**Pull request merged**